templates/node--localgov-news-article--full.html.twig

```diff
@@ -67,23 +66,41 @@
  */
 #}
 
-<article{{ content_attributes.addClass('content').addClass('newsroom--article') }}>
-  <div>
-    <div class="newsroom--info">
-      <span class="newsroom--date">{{ node.localgov_news_date()|date("j F Y") }}</span>
+{%
+  set classes = [
+  'news-article',
+  'node',
+  'node--type-' ~ node.bundle|clean_class,
+  node.isPromoted() ? 'node--promoted',
+  node.isSticky() ? 'node--sticky',
+  not node.isPublished() ? 'node--unpublished',
+  view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+]
+%}
 
-      {% if not node.localgov_news_categories.isEmpty() %}
-        <span class="newsroom--category">
-        <a href="{{ path('entity.taxonomy_term.canonical', {'taxonomy_term': node.localgov_news_categories.entity.id()}) }}">
-          {{ node.localgov_news_categories.entity.label() }}
-        </a>
-      </span>
-      {% endif %}
+<article{{ attributes.addClass(classes) }}>
 
-    </div>
+  <div class="news-article__metadata">
+    <time datetime="{{ node.localgov_news_date()|date("U") }}" class="news-article__metadata-item news-article__metadata--date">
+      {{ node.localgov_news_date()|date("j F Y") }}
+    </time>
+    {% if not node.localgov_news_categories.isEmpty() %}
+      <ul class="news-article__metadata-item news-article__metadata--categories" aria-label="Article categories">
+        {% for key,value in node.localgov_news_categories.value %}
+          <li>
+            {{ content.localgov_news_categories[key] }}
+          </li>
+        {% endfor %}
+      </ul>
+    {% endif %}
+  </div>
+
+  <h1{{ title_attributes.addClass('news-article__title') }}>
```

Comment:
I left this item in in my PR, but we have to be careful about it as the page title is already printed in the page title block, so might appear twice in some pages. 

My preference is to leave it here and to _not_ use the page title block for node pages. That might mean updating the config to only show the page title block on some content types, or (more sustainably) to put a conditional around the page title region in `page.html.twig` to not load the page title if the page type is `node`.

Reply:
I've been suppressing the page title block on the London Councils site to overcome this which isn't ideal.

It's a bit bound up in the theme page structure, where the page title block and therefore the page heading currently sits outside of any meaningful container. Discussion here: https://github.com/localgovdrupal/localgov_theme/issues/149

Reply:
It's a Drupal hangover. I really don't like it. I'd much prefer if we had the page titles printed in the node templates.

In my theme in work (not for LocalGov), I have this in the page.html.twig file:

```
{% if not node %}
  {{ page.title }}
{% endif %}
```

And that ensures that the page block is not printed on any node pages. So there is no config overrides needed per content type. That's an approach we could pursue here. Or perhaps add it as a theme setting on the theme's setting pages, to make sure we don't break any current sites.

Reply:
That approach means we need to maintain that code and keep it updated if
- **we** add any new content types, and/or
- **any council** using localgovdrupal adds any new content types (which will mean custom modules to override custom modules)

I'd probably prefer to have it set that:
- Page Title block does not display on any node page
- We have a `node--full.html.twig` as the default for all nodes that don't have a more specific template (such as `node--news-article--full.html.twig`) and print the page title in that template

That is a more scalable approach I think, rather than something that needs maintenance and re-thinking about each time new things are added.

Reply:
Understood, that makes a lot of sense for future maintenance.

Is there any way we can achieve it in this module within the existing bounds of the core module and theme?

Reply:
You idea to follow the approach of the subsites module would work, but then it would mean we have _some_ code hiding the page title block in the news module and _some_ code hiding the page title block in the subsites module. If events or some other module wants to do the same thing, you can see how it will start getting problematic at scale to have the same function spread across multiple modules.

My suggestion in the short term might be to add news as one of the content types in the list in the subsites module, then create an issue for us to tackle this is a sustainable manner - in localgov_core or localgov_skeleton, or somewhere else. I don't think this is a hard issue to solve, but it might take some coordinating with the other councils if it's removing a block that they are expecting.

Reply:
I agree with the approach, but I'm not keen on the suggested short term solution - we'd have to make the subsites module as a dependency for the news module if we add the news content types there, which feels messier than using the mechanism that localgov_core provides for this purpose.

I've added an event subscriber to news for now: https://github.com/localgovdrupal/localgov_news/pull/15/commits/4bc769e4ca9e49ca9338c16569f51ccc2defa446

This lets us get closer to a release for news and can be removed later.